tests/integration/testdata/buildcmd/Provided/Makefile-container

```diff
@@ -2,6 +2,6 @@ build-Function:
 	cp *.py $(ARTIFACTS_DIR)
 	cp requirements.txt $(ARTIFACTS_DIR)
 	curl https://bootstrap.pypa.io/get-pip.py > /tmp/get-pip.py
```

Comment:
Shouldn't this be changed to `https://bootstrap.pypa.io/2.7/get-pip.py` instead?

See https://github.com/pypa/get-pip/pull/81 and the similar instruction for Python 2.6: https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py

Reply:
@aahung and I had a discussion about this. We think it's safer to assume `python3` exists than to assume `python` always runs the Python2 Interpreter. 
If we change to https://bootstrap.pypa.io/2.7/get-pip.py, we need to assume `python` is always running Python2

Reply:
in fact, we verified that both work

Reply:
I see, if this isn't testing anything related to Python 2 then it seems fine.

Reply:
yeah, this one is not related to Python2.
We also need to fix py2.7 build image though.

Reply:
^ #2561